constants.py

```diff
@@ -1,6 +1,7 @@
 #IPE assignment naming convention:
 ASSIGNMENT_NAME = 'IPE Competencies Attained'
 ASSIGNMENT_GROUP_NAME ='IPE Competencies'
+ASSIGNMENT_DESCRIPTION = 'This assignment was generated by the Center for IPE to track student attainment of IPE competencies. It does not impact students\' grades. So please do not modify or remove it. If you have questions, please contact IPECenter@umich.edu.'
```

Comment:
Maybe use the full official name for IPE center as [the Michigan Center for Interprofessional Education](https://interprofessional.umich.edu/the-center/directors-welcome/)

Reply:
Change "So please do not modify or remove it."  to "Please do not modify or remove this assignment. ". Maybe the latter sounds more formal?

Reply:
That Description is provided by IPE Leads teams. So @jennlove-um thoughts? for changes

Reply:
Then don't worry since this is provided by IPE leads team.